src/libs/actions/Report.js

```diff
@@ -183,7 +183,7 @@ function getSimplifiedReportObject(report) {
     const createTimestamp = lodashGet(report, 'lastActionCreated', 0);
     const lastMessageTimestamp = moment.utc(createTimestamp).unix();
     const lastActionMessage = lodashGet(report, ['lastActionMessage', 'html'], '');
-    const isLastMessageAttachment = /<img([^>]+)\/>/gi.test(lastActionMessage);
+    const isLastMessageAttachment = /<img|a\s[^>]*data-expensify-source\s*=\s*"[^"]*"[^>]*>/gi.test(lastActionMessage);
```

Comment:
**suggestion:** add a constant for `data-expensify-source`. There are a few usages in the codebase and it the code would be clearer if we used a constant that said what `data-expensify-source` is vs. directly reference it.

something like `CONST.ATTACHMENT_ATTRIBUTE_SOURCE` maybe?

Reply:
@dennismunene Let us know if you need any help with this. 

Great suggestion!

Reply:
@mountiny Yes would appreciate some help with this ,  do i need to raise a PR here https://github.com/Expensify/expensify-common to introduce the new CONST ?

Reply:
@dennismunene Nono, I think adding the const [here](https://github.com/Expensify/App/blob/ee5d18e2bb47a9d2b71fcca08ae1254b56a95d4c/src/CONST.js#L11-L657) and then importing it _should_ be sufficient :) 

We should then look into a follow up to refactor the other places where there is a use of `data-expensify-source` to update it, but we do not have to do it in this PR!

Reply:
Great , thank you 🙂

Reply:
Pushed update with this suggestion. 

What is the correct flow to follow up on the refactor task after this PR is merged ?